ui/app/AppLayouts/Chat/controls/Contact.qml

```diff
@@ -54,7 +54,7 @@ Rectangle {
         id: usernameText
         text: name
         elide: Text.ElideRight
-        anchors.right: parent.right
+        anchors.right: assetCheck.left
```

Comment:
This should be conditional. If the check is visible, use the check, if not, use parent.

Reply:
agreed :)